ui/test/framework/stf/src/saros/stf/server/rmi/htmlbot/widget/impl/RemoteHTMLMultiSelect.java

```diff
@@ -31,9 +30,7 @@ public static RemoteHTMLMultiSelect getInstance() {
 
   @Override
   public void select(List<String> value) throws RemoteException {
-    String name = BotUtils.getSelectorName(selector);
-
-    browser.syncRun(String.format("view.setFieldValue('%s', '%s')", name, value.toString()));
+    new JQueryHelper(browser).setFieldValue(selector, value.toString());
```

Comment:
is it possible not to create a new instance of the JQueryHelper every time? it feels like a waste.

maybe make the browser a parameter in JQueryHelper? or maybe create an instance of JQueryHelper for every instance of browser and attach it directly to the browser?

Reply:
I will move the helper instance into the abstract class `HTMLSTFRemoteObject`